train.py

```diff
@@ -481,12 +494,37 @@ def main(args):
 			onnxruntime.set_default_logger_severity(0)
```

Comment:
Please support passing severity in the Wrapper's constructor

Reply:
done

Reply:
Have you tested if severity will get set with `--verbose`? I think it currently has a bug

Reply:
Fix a bug.